packages/web/src/pages/sign-up-page/SignUpPage.tsx

```diff
@@ -102,6 +102,7 @@ export const SignUpPage = () => {
         <SignUpRoute exact path={SIGN_UP_COMPLETED_REDIRECT}>
           <Redirect to={FEED_PAGE} />
         </SignUpRoute>
+        <SignUpRoute path='*' />
```

Comment:
we had logic to redirect from invalid routes but they don't ever trigger since the logic was inside SignUpRoute